src/App.css

```diff
@@ -145,6 +145,34 @@ header p {
   color: #2d3748;
 }
 
+.result-card .subtitle {
+  font-size: 0.9rem;
+  color: #666;
+  margin-top: 0.5rem;
```

Comment:
🔰 **Junior Developer**

Love the new `.result-card.highlight` CSS! But, the color `#f0f8ff` is a bit light against the default background (App.css line 151). On some monitors, the contrast might not be strong enough for accessibility. Can we bump up the contrast or check it against WCAG guidelines?